build.gradle

```diff
@@ -157,4 +157,31 @@ def getProjectVersion() {
         ver = versionLabel + "-" + count + "." + commitId + "-SNAPSHOT"
     }
     return ver
-}
\ No newline at end of file
+}
+
+distributions {
+    release {
+        contents {
+            from shadowJar
+            from(project.configurations.shadow)
+            from javadocJar
+            from sourceJar
+        }
+    }
+    workspace {
```

Comment:
What is this 'workspace' distribution about?

Reply:
Why the word 'workspace' as opposed to, say, 'release'?

Reply:
It is named `workspace` to indicate that we are packaging the entire project workspace. The actual assembly/packaging will have the appropriate name meant for distribution.